docs/content/guides/7.multistore/3.patterns/5.subpath/1.subpath.md

```diff
@@ -0,0 +1,696 @@
+
+3. **Same-Store Operation**
+    - Config Switcher works within a single store instance
+    - It extracts the configuration ID from the request
+    - It executes requests using the appropriate configuration
+
+::tip Config Switcher Documentation
+This section provides a brief overview of setting up Config Switcher. For complete documentation, refer to the [Config Switcher guide](https://docs.alokai.com/middleware/guides/config-switcher).
+::
+
+### Key Distinctions To File-Based Inheritance
+
+`ConfigSwitcher` operates differently from file-based inheritance:
+
+| **File-Based Inheritance**              | **Config Switcher**                                              |
+| --------------------------------------- | ---------------------------------------------------------------- |
+| Works **between different stores**      | Works **within a single store**                                  |
+| Creates separate deployable instances   | Uses a single deployable instance                                |
+| Full codebase customization             | Only middleware configuration changes, Storefront code is shared |
+| Requires separate builds for each store | Single build supports multiple configurations                    |
+
+Take a look at the diagram to see the difference:
+
+<img src="../../img/file-based-vs-switcher-diagram.png" alt="Comparison of File-based Inheritance vs Config Switcher" class="mx-auto">
+
+## Setting Up the Middleware
+
+The first step is to configure your Middleware to support different configurations using the Config Switcher extension.
+
+Create a Config Switcher extension for your integration:
+
+```ts
+// apps/storefront-middleware/integrations/<integration-name>/extensions/configSwitcher.ts
+import { createConfigSwitcherExtension } from '@alokai/connect/config-switcher';
+import type { MiddlewareConfig } from '@vsf-enterprise/<integration>-api';
+
+export const configSwitcherExtension = createConfigSwitcherExtension<MiddlewareConfig>({
+  // Config switcher options - specify only what's different for each store
+  configuration: {
+    apparel: {
+      api: {
+        baseSiteId: 'apparel',
+        catalogId: 'apparelProductCatalog',
+      },
+    },
+    electronics: {
+      api: {
+        baseSiteId: 'electronics',
+        catalogId: 'electronicsProductCatalog',
+      },
+    },
+  },
+});
+```
+
+Then add this extension to your integration configuration:
+
+```ts
+// apps/storefront-middleware/integrations/<integration-name>/config.ts
+import { configSwitcherExtension } from './extensions/configSwitcher';
+
+export default {
+  integrations: {
+    <integration-name>: {
+      // Base configuration shared across all stores
+      configuration: {
+        // ...
+      },
+      extensions: (predefinedExtensions) => [
+        ...predefinedExtensions,
+        configSwitcherExtension,
+      ]
+    }
+  }
+};
+```
+
+## Customizing Store Appearance with CSS Variables
```

Comment:
Please add a complete example on how to do it

Reply:
done https://github.com/vuestorefront/vue-storefront/pull/7422/commits/c8e0ca6d5af95e822a75a0d9dcc736f604b215f3